### ISSUE TITLE
Fehler bei Dokumentenimport mit Umlauten im Namen

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-client#137]
Ich habe versucht eine Officedatei (Libreoffice writer) zu importieren, als .odt, als docx..., die einen Umlaut (ä) im Namen hat (verkneife ich mir normalerweise, weil Umlaute international nicht bekannt sind. Das ging wie folgt daneben: Die Datei wird importiert, der Umlaut im Namen als Fragezeichen angezeigt, Datei soll dann aber nicht vorhanden sein und lässt sich nicht öffnen, keine Vorschau. Der Eintrag lässt sich aus dem Dokumentenordner auch nicht mehr löschen (Datei existiert nicht).... Wenn man den Umlaut vor dem Import umbenennt, geht alles normal mit odt und docx. Gretchenfrage, wie bekommt man die toten Einträge weg....

**Comment by j-dimension** [comment]
Wie genau wurde "importiert"? Per Drag & Drop in die Akte gezogen? 

Umlautproblem können eigentlich nur dann auftreten, wenn Dateien zwischen verschiedenen Betriebssystemen via ZIP transferiert werden. Dieser Fall hier wäre mir also neu.

Bitte mal http://j-lawyer-box öffnen und unter "Operationen" den Punkt "Dateinamen prüfen" ausführen - erscheinen dort Dokumente in der Ausgabe?

**Comment by Mucki526** [comment]
Nein, ich habe die Datei geschrieben und dann versucht zu importieren über den Einfügen-Button. Mit Drag & Drop habe ich noch garnicht gearbeitet. Danke für den Tipp....  Nur wie bekomme ich die toten Einträge wieder raus....??? (Wenn nicht bleiben sie drinnen, hoffe die machen keinen Ärger....)

**Comment by j-dimension** [comment]
Danke für die Rückmeldung. Was hat das hier ergeben?
-- Bitte mal http://j-lawyer-box öffnen und unter "Operationen" den Punkt "Dateinamen prüfen" ausführen - erscheinen dort Dokumente in der Ausgabe?

Ggf. machen wir einen kurzen Termin via Fernwartung. Würde auch gern verstehen was dort genau passiert.
